int64 status
int64 x_tujuan
int64 y_tujuan